### PR TITLE
Use new version features

### PR DESCRIPTION
**Pull request by gkoz**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -19,7 +16,7 @@ git = "https://github.com/gtk-rs/gdk"
 git = "https://github.com/gtk-rs/gtk"
 
 [features]
-gtk_3_10 = []
+gtk_3_10 = ["gtk/3.10"]
 gtk_3_14 = ["gtk_3_10"]
```
Comment (EPashkin) [inline Cargo.toml]: This don't need "gtk/3.14" for completeness sake? Or to be removed as unused?

Comment (gkoz) [inline Cargo.toml]: Ideally `gtk_3_14` should not be there, I put it in when branching appveyor into 3.6 and 3.14 to try avoiding some confusion. Now it's all confusing anyway so might as well remove 3_14 but it has to start with CI scripts changes.

**Comment by gkoz** [comment]
Looks like the version features merge fallout is fixed now.

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**